fuzzers/030-iob/Makefile

```diff
@@ -13,7 +13,8 @@ build/segbits_xiob33.rdb: $(SPECIMENS_OK)
 
 build/segbits_xiob33.db: build/segbits_xiob33.rdb process_rdb.py bits.dbf
 	python3 process_rdb.py build/segbits_xiob33.rdb > build/segbits_xiob33_processed.rdb
-	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in build/segbits_xiob33_processed.rdb --seg-fn-out $@
+	python3 group.py -g tag_groups.txt -i build/segbits_xiob33_processed.rdb -o build/segbits_xiob33_processed.rdb2
+	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in build/segbits_xiob33_processed.rdb2 --seg-fn-out $@
```

Comment:
Should the feature set of `group.py` be part of `XRAY_DBFIXUP`?

Reply:
I moved the `group.py` to the `utils` folder. Do you want this tool to be a part of the dbfixup script? At first I wanted to be that way but I found it difficult to implement the functionality in the dbfixup script architecture.

Reply:
Is there a way to have it be a part of the dbfixup script, but only in the lightest sense?  Basically don't try to integrate it complete, but make it an isolated pass within the script?

The idea is future fuzzers might use either the zero-db or the group style?  Thoughts?

Reply:
Hmmm, Ok. I can make it as an additional pass to the db_fixup. So my idea is to keep `group.py` as a separate script but call it from within the db_fixup.